docs/api/effector/babel-plugin.md

```diff
@@ -88,150 +88,151 @@ getUsers.use(
 
 ## Configuration
 
-### noDefaults
+### importName
+
+- Type: `string | string[]`
+- Default: `['effector', 'effector/compat']`
 
-> `noDefaults: boolean`
+Specifying import name or names to process by plugin. Import should be used in the code as specifed.
+
+### factories
 
 :::note since
-effector 20.2.0
+effector 21.6.0
 :::
 
-Option for effector/babel-plugin for making custom unit factories with clean configuration.
+- Type: `string[]`
+
+Accepts an array of module names which exports treat as custom factories therefore each function call provides unique prefix for [sids](./babel-plugin.md#sid) of units inside them. Used to SSR([Server Side Rendering](https://developers.google.com/web/updates/2019/02/rendering-on-the-web)) and it's not required for client-only application.
```

Comment:
I suggest to mention tests here. Or replace mention of SSR by general mentions of Fork API.